src/System.Net.Http/tests/UnitTests/Fakes/HttpClientHandler.cs

```diff
@@ -0,0 +1,132 @@
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public DecompressionMethods AutomaticDecompression
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool UseProxy
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public IWebProxy Proxy
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool PreAuthenticate
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool UseDefaultCredentials
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public ICredentials Credentials
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool AllowAutoRedirect
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public int MaxAutomaticRedirections
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public long MaxRequestContentBufferSize
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        #endregion Properties
+
+        #region De/Constructors
+
+        public HttpClientHandler()
+        {
+            throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented");
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+        }
+
+        #endregion De/Constructors
+
+        #region Request Execution
+
+#pragma warning disable 1998 // async method with no await
```

Comment:
Instead of suppressing the warning, you could just remove the 'async' keyword from the method.